mfeesm_connector.py

```diff
@@ -46,7 +44,7 @@
 }
 
 
-class MFENitroConnector(BaseConnector):
+class MFEEsmConnector(BaseConnector):
```

Comment:
we have decided to change the name of repo since app-name was mcafeeesm but repo name was mcafeenitro, so it was causing bit confusion. based on new name also renamed the class name accordingly.
link of conversation = https://splunk.slack.com/archives/C01F1PWD3S6/p1644929860890719